docs_user/modules/proc_performing-a-fast-forward-upgrade-on-compute-services.adoc

```diff
@@ -62,17 +62,17 @@ spec:
 '
 ----
 
-. Wait for Compute control plane services' CRs to be ready:
+. Wait for the Compute control plane services CRs to be ready:
 +
 ----
-oc wait --for condition=Ready --timeout=300s Nova/nova
+$ oc wait --for condition=Ready --timeout=300s Nova/nova
```

Comment:
Is Nova/nova the name of a Compute service? Should Nova/nova be replaced with a user replacement value? e.g.,(<compute_service>)

Reply:
no, those are "hardcoded" k8s API name for Nova CRDs and a 'nova' CR instance of it. @gibizer am I right?